ona/models.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.postgres.fields import HStoreField
+from django.db import models
+from accounts.models import Integration
+from reports.models import Report
+
+
+class Submission(models.Model):
+
+    integration = models.ForeignKey(Integration,
+                                    related_name='submissions',
+                                    null=False)
+    report = models.ForeignKey(Report,
+                               related_name='submissions',
+                               null=True)
+    content = models.TextField(null=False, blank=False)
+    metadata = HStoreField(null=True, blank=True, default={})
+    created_at = models.DateTimeField(auto_now_add=True)
+    submitted = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.content
+
+# send new submissions
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .tasks import send_submission
+
+
+@receiver(post_save, sender=Submission)
```

Comment:
It might be a good idea to add `dispatch_uid='ona.post_save.submission'` or similar in order to prevent the duplicate signal handlers being connected in odd situations (not very likely, but it is polite and is occassionally useful in other situations like disconnecting signals in tests). Largely just mentioning it in case you hadn't seen it in the docs.